class/defaults.yml

```diff
@@ -1,9 +1,10 @@
 parameters:
   nfs_subdir_external_provisioner:
     namespace: syn-nfs-subdir-external-provisioner
+    multi_instance: true
```

Comment:
Just setting `multi_instance: true` is not sufficient to actually make the component multi-instance. You have to explicitly ensure that instantiating the component multiple times will generate distinct files, primarily in `class/nfs-subdir-external-provisioner.yml`. You can use `${_instance}` to ensure outputs are placed in different directories based on the instance identifier.

See https://syn.tools/commodore/reference/architecture.html#_component_instantiation for a detailed explanation of what Commodore does, and what's the responsibility of the component.

Reply:
Added instance identifier to create distinct output directories.